spec/lib/census_api_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
-describe CensusApi do
-  let(:api) { described_class.new }
+describe 'CensusApi' do
```

Comment:
I don't understand the reason for commit 424dfdf5. Any ideas?

Reply:
Sounds good to me! :ok_hand: